WebContent/WEB-INF/jsp/dataSourceEdit/editModbus.jsp

```diff
@@ -283,7 +283,9 @@
     <c:set var="dsHelpId" value="modbusIpDS"/>
   </c:when>
 </c:choose>
+<div style="width: 70%">
```

Comment:
It's better than it was, but with higher magnification the problem still persists:
![image](https://github.com/SCADA-LTS/Scada-LTS/assets/35842300/fe15a62f-d527-4893-97b0-b79d023d0299)

I suggest going back and modifying the pointList.tag file (position: fixed to position: relative, width: 30% to width: 100%) :
`<div id="pointDetails" class="borderDiv marB" style="display: none; width: 30%;position: fixed; right: 0;">`
on:
`<div id="pointDetails" class="borderDiv marB" style="display: none; width: 100%;position: relative; right: 0;">`